src/lib.rs

```diff
@@ -217,13 +219,14 @@ impl TrowBuilder {
             .address(self.config.addr.host.clone())
             .port(self.config.addr.port)
             .keep_alive(0) // Needed to mitigate #24. See above.
-            .secret_key(skey)
+            .secret_key(secret_key)
             .workers(256);
 
         if let Some(ref tls) = self.config.tls {
             if !(Path::new(&tls.cert_file).is_file() && Path::new(&tls.key_file).is_file()) {
                 return  Err(format_err!("Trow requires a TLS certificate and key, but failed to find them. \nExpected to find TLS certificate at {} and key at {}", tls.cert_file, tls.key_file));
             }
+            println!("{} - {}", tls.cert_file.clone(), tls.key_file.clone());
```

Comment:
Did you mean to delete this?

Reply:
Yep same as above. Was debugging the TLS part. It's a left over. Good catch. Removed it.